### PR TITLE
zigbee: Add new known issues for NCS v1.6.0

### PR DESCRIPTION
**Pull request by tomchy**
Add new issues, found between NCS releases.

Signed-off-by: Tomasz Chyrowicz <tomasz.chyrowicz@nordicsemi.no>

### CODE REVIEW

**Comment by tomchy** [comment]
This PR includes new known issues, extracted from https://github.com/nrfconnect/sdk-nrf/pull/4840

**Review by maciejpietras — APPROVED** [review]

**Review by sebastiandraus — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -303,6 +303,25 @@ KRKNWK-6408: ``diag`` command not supported
...
+
+.. rst-class:: v1-6-0
+
+KRKNWK-9743 Timer can't be stopped in Zigbee routers and coordinators
+  The call to the ``zb_timer_enable_stop()`` API has no effect on the timer logic in Zigbee routers and coordinators.
+
+.. rst-class:: v1-6-0
+
+KRKNWK-10490: Dead lock in NCP frame fragmentation logic
+  If the last piece of a fragmented NCP command is not delivered, the receiving side becomes unresponsive to further commands.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:323]:
```suggestion
KRKNWK-10490: Deadlock in the NCP frame fragmentation logic
  If the last piece of a fragmented NCP command is not delivered, the receiving side becomes unresponsive to commands.
```
No workaround, right?

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Comment by greg-fer** [comment]
Some of my comments are marked as `Outdated`, so please check them in the Conversation tab, not in the Files changed tab.

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (tomchy) [reply doc/nrf/known_issues.rst:323]: Yes, there is no workaround for this issue.

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -303,6 +303,25 @@ KRKNWK-6408: ``diag`` command not supported
...
+
+.. rst-class:: v1-6-0
+
+KRKNWK-9714: Device association fail if the Request Key packet is retransmitted
+  If the Request Key for the TCLK is retransmitted and the coordinator sends two, different new keys, it confuses the joiner's logic and lead to unsuccessful key verification.
+
+.. rst-class:: v1-6-0
+
+KRKNWK-9743 Timer can't be stopped in Zigbee routers and coordinators
+  The call to the`` zb_timer_enable_stop()`` API has no effect on the timer logic in Zigbee routers and coordinators.
```
[the remark this answers is not in the document]
Comment (tomchy) [reply doc/nrf/known_issues.rst]: Yes, there is no workaround for this issue.

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -303,6 +303,25 @@ KRKNWK-6408: ``diag`` command not supported
...
 ======
 
+.. rst-class:: v1-6-0
+
+KRKNWK-8211: The :c:macro:`ZB_ZDO_SIGNAL_LEAVE signal` is generated twice during ZC factory reset
+
+.. rst-class:: v1-6-0
+
+KRKNWK-9714: Device association fail if the Request Key packet is retransmitted
+  If the Request Key for the TCLK is retransmitted and the coordinator sends two, different new keys, it confuses the joiner's logic and lead to unsuccessful key verification.
```
[the remark this answers is not in the document]
Comment (tomchy) [reply doc/nrf/known_issues.rst:313]: Yes, there is no workaround for this issue.

File: doc/nrf/known_issues.rst [hunk 4]
```diff
@@ -303,6 +303,25 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
+.. rst-class:: v1-6-0
+
+KRKNWK-8211: The :c:macro:`ZB_ZDO_SIGNAL_LEAVE signal` is generated twice during ZC factory reset
+
```
[the remark this answers is not in the document]
Comment (tomchy) [reply doc/nrf/known_issues.rst]: ZC replaced with "Zigbee Coordinator". It is possible to implement a workaround for it, but we have not created nor tested one.

**Review by greg-fer — APPROVED** [review]

**Review by ioannisg — APPROVED** [review]

**Pull request merged**